.stylelintrc.json

```diff
@@ -1,36 +1,30 @@
 {
-  "ignoreFiles": ["**/*"],
-  "overrides": [
-    {
-      "files": ["**/*.css"],
-      "extends": ["stylelint-config-standard", "stylelint-config-css-modules"],
-      "plugins": ["stylelint-order"],
-      "rules": {
-        "order/order": ["custom-properties", "declarations"],
-        "order/properties-order": [
-          ["composes"],
-          {
-            "unspecified": "bottomAlphabetical"
-          }
-        ],
-        "property-disallowed-list": [
-          [
-            "margin",
-            "margin-bottom",
-            "margin-left",
-            "margin-right",
-            "margin-top",
-            "padding",
-            "padding-bottom",
-            "padding-left",
-            "padding-right",
-            "padding-top"
-          ]
-        ],
-        "selector-class-pattern": null,
-        "unit-disallowed-list": ["px"]
+  "extends": ["stylelint-config-standard", "stylelint-config-css-modules"],
+  "plugins": ["stylelint-order"],
+  "rules": {
+    "media-feature-range-notation": "prefix",
+    "order/order": ["custom-properties", "declarations"],
+    "order/properties-order": [
+      ["composes"],
+      {
+        "unspecified": "bottomAlphabetical"
       }
-    }
-  ],
-  "rules": {}
+    ],
+    "property-disallowed-list": [
+      [
+        "margin",
```

Comment:
Wat is eigenlijk de reden dat deze properties disallowed zijn?
Hoe kan ik zelf handmatig padding/margin setten van bepaalde elementen?

Reply:
Ik snap dat dit niet related is aan de PR maar ik was toch  even benieuwd!

Reply:
`margin` and `padding` are a shorthand for related directional properties, in other words `*-top`, `*-right`, `*-bottom`, `*-left`. 

In our codebase we use [logical properties](https://developer.mozilla.org/en-US/docs/Web/CSS/CSS_logical_properties_and_values), such as `padding-block` and `margin-inline-end`. This allows us to support languages that are not left-to-right.